etc/deltaDebug.py

```diff
@@ -203,7 +204,7 @@ def debug(self):
     # and calls the step function, then returns the stderr of the step.
     def perform_step(self, cut_index=-1):
         # read base db in memory
-        self.base_db = odb.dbDatabase.create()
+        self.base_db = Design.createDetachedDb()
```

Comment:
What is the significance of this change?

Reply:
Logger handling